packages/slice-machine/src/legacy/lib/utils/preferWroomBase.ts

```diff
@@ -1,13 +1,23 @@
 export default function preferWroomBase(smApiUrl: string): string {
```

Comment:
I kept this function as its legacy and a hotfix, but this should be reworked totally, and we should ask the manager for the correct login URL as the manager already handles that.